### ISSUE TITLE
Soccer arenas are classified as "tracks" in add-on menu

### ISSUE THREAD

**Issue by Wuzzy2** [supertuxkart/stk-code#4410]
## Description

In the add-ons menu, all soccer arenas are classified as “tracks” (rather than arenas). This makes them harder to find.

## Version

STK 1.2

**Comment by STK-FabianF** [comment]
There should be an own category for soccer fields but this is more of an issue in stkaddons?

**Comment by Wuzzy2** [comment]
It's probably both. If stkaddons is updated, the STK software needs to be updated as well (for the new tab), right?

**Comment by Benau** [comment]
need some thinking on how to do that as it breaks backward compatibility ![](https://cdn.lihkg.com/assets/faces/pig/sowhat.gif)

**Comment by Alayan-stk-2** [comment]
I have been thinking for a while now that soccer fields shouldn't be put under tracks. But I don't think putting them under arena is such a great idea either, rather they should get their own category. This would be the best to find faster a track/arena/soccer field in the addons list.

**Comment by GladiosAnimations** [comment]
yeah. when the soccer will get its own category, it will be better.

**Comment by kimden** [comment]
Also some soccer fields are classified as "arenas" because FFA/CTF also can be ticked for the map, and they are only "tracks" when there's none of battle modes.

Probably it also makes sense to put all arenas where FFA/CTF can be played into one category, all arenas supporting soccer into another, etc. - while those sets might intersect, it better suits people who are searching maps for a single mode. (Technically, CTF is independent from FFA and is supported on OMC for example - should they be different categories? I'd rather say no)
